src/config/featureConfiguration.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "index": 9999,
+    "internalName": "INTERNAL_TESTING_ENABLED_BY_DEFAULT",
```

Comment:
I tested this by building the plugin, and confirming that there were no changes in the Settings dialog.

I edited an existing setting and saw that the following was added to my `data.json` file:

```
  "features": {
    "INTERNAL_TESTING_ENABLED_BY_DEFAULT": true
  }
```

I think we will get some questions from users asking what testing it is enabling? And trying to disable it.

I think it's kind of a dummy placeholder feature. Perhaps something like `DUMMY_PLACEHOLDER_FEATURE` or `CODE_VALIDATION_FEATURE` (based on the description) might be a clearer name?

Reply:
I've just realised that if `INTERNAL_TESTING_ENABLED_BY_DEFAULT` is renamed and then that change is merged back to tasks-x it will likely cause confusion or real problems for users that have previously tasks-x and saved any settings.

So it's definitely not worth doing.

Marking conversation as resolved.